casper/src/test/scala/coop/rchain/casper/genesis/contracts/TestSetUtil.scala

```diff
@@ -22,10 +24,12 @@ object TestSetUtil {
     Runtime.create(storageLocation, size)(taskCapture)
   }
 
-  def eval_term(term: Par, runtime: Runtime)(implicit scheduler: Scheduler): Unit =
+  def eval_term(term: Par, runtime: Runtime)(implicit scheduler: Scheduler,
+                                             rand: Blake2b512Random): Unit =
     runtime.reducer.inj(term).unsafeRunSync
 
-  def eval(code: String, runtime: Runtime)(implicit scheduler: Scheduler): Unit =
+  def eval(code: String, runtime: Runtime)(implicit scheduler: Scheduler,
+                                           rand: Blake2b512Random): Unit =
```

Comment:
Since you are splitting and explicitly wiring it in several places, you might consider making `rand` an explicit parameter for these helpers.

Reply:
I'll look at these again.